front/pages/w/[wId]/u/chat/[cId]/index.tsx

```diff
@@ -919,6 +966,26 @@ export default function AppChat({
     })();
     setLoading(false);
   };
+  const handleFeedback = (
+    message: ChatMessageType,
+    feedback: MessageFeedbackStatus
+  ) => {
+    setMessages(
+      messages.map((m) => {
+        if (m === message) {
+          m.feedback = feedback;
+        }
+        return m;
+      })
+    );
+    const chatUrl = `https://dust.tt/w/${owner.sId}/u/chat/${chatSession.sId}`;
+    logger.info({
```

Comment:
Logger does not function from client side (it'll just write to console.log)

We'll want to add logger + statsd increment in the endpoints that will be called from here once this is stateful.

You can already create this endpoint for logger and statsd purposes before we even add the models :+1: